app/assets/javascripts/discourse/controllers/bookmark.js

```diff
@@ -49,7 +84,12 @@ export default Controller.extend(ModalFunctionality, {
       userTimezone: this.currentUser.resolvedTimezone()
     });
 
+    this.bindKeyboardShortcuts();
     this.loadLastUsedCustomReminderDatetime();
+
+    // make sure the input is cleared, otherwise the keyboard shortcut to toggle
+    // bookmark for post ends up in the input
+    setTimeout(() => this.set("name", null));
```

Comment:
Also why do we need this ? :p and not just `this.set("name", null)`

I guess you tried and it wasn't working, do you have an explanation of why ?

Reply:
@jjaffeux the reason I need to do this is because if you e.g. press j and then b to bookmark a post, the input gets the `b` from the shortcut filled in. Will use `Ember.run.next` here tho! :+1: (if there is another way to avoid this I would like to know)

![peek](https://user-images.githubusercontent.com/920448/78521540-846f0e00-780d-11ea-8a6a-662983f4181c.gif)